include/boost/sml/aux_/utility.hpp

```diff
@@ -299,6 +299,15 @@ struct zero_wrapper<R (TBase::*)(TArgs...) const> {
   R (TBase::*ptr)(TArgs...) const {};
 };
 
+template <class R, class... TArgs>
+struct zero_wrapper<R (*)(TArgs...)> {
```

Comment:
Can we add test case/example to verify/illustrate the usage and the use case :pray: :white_check_mark: thanks

Reply:
I modified an example to show how this can be used. It's basically to avoid creating lambdas to forward the call to the actual function when it's not necessary. This leads to less traction when using functions that have their definition outside of headers.